runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -46,6 +51,24 @@ def __init__(
         self.alibi_explain_settings = explainer_settings
         super().__init__(settings)
 
+    @custom_handler(rest_path="/explain_v1_output")
```

Comment:
This coudl actually just be changed to be `/explain`, as in theory it would then align with the path that the v1 sever would use (as opposed to the actual v2 compliant which would be `/.../infer`

Reply:
done